CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/filechooser/FileChooser.java

```diff
@@ -133,20 +97,9 @@ public static CompletableFuture<Optional<List<File>>> openMultiDialog(final File
      * @param fileDialogMode the type of file chooser dialog to open, save, open or multi
      * @return the selected file(s) or an empty optional if the user selects cancel
      */
-    private static Optional<List<File>> openFileDialog(final FileChooserBuilder fileChooserBuilder,
-                                                       final FileChooserMode fileDialogMode) {
-        final ShowFileChooserDialog showDialog = new ShowFileChooserDialog(
-                fileChooserBuilder, fileDialogMode);
-        
-        try {
-            // Make a request to open the file chooser dialog on the UI thread
-            EventQueue.invokeAndWait(showDialog);
-        } catch (final InterruptedException ex) {
-            LOGGER.log(Level.WARNING, "Thread displaying the file chooser was interrupted.", ex);
-            Thread.currentThread().interrupt();
-        } catch (final InvocationTargetException ex) {
-            LOGGER.log(Level.SEVERE, "Error occurred during selection in file chooser.", ex);
-        }
+    private static Optional<List<File>> openFileDialog(final FileChooserBuilder fileChooserBuilder, final FileChooserMode fileDialogMode) {
+        final ShowFileChooserDialog showDialog = new ShowFileChooserDialog(fileChooserBuilder, fileDialogMode);
+        showDialog.run();
```

Comment:
I suspect this will need to have a check here ...
If it's not on the EventDispatchThread, then it will still need to do the invokeAndWait() call, instead of directly call the run() method.